test/unit/printer/layers.test.js

```diff
@@ -167,25 +167,25 @@ describe('layer creation', () => {
     });
 
     it('initially emit a status with progress 0', () => {
-      expect(received).toEqual([0, null]);
+      expect(received).toEqual([0, null, undefined]);
```

Comment:
Is it mandatory to explicitly add `undefined` ?

Reply:
Currently, yes I think, could perhaps make it optional

Reply:
Made it optional